includes/class-affiliatewp-affiliate-area-shortcodes.php

```diff
@@ -0,0 +1,253 @@
+<?php
+/**
+ * Core: Plugin Bootstrap
+ *
+ * @package     AffiliateWP Affiliate Area Shortcodes
+ * @subpackage  Core
+ * @copyright   Copyright (c) 2021, Sandhills Development, LLC
+ * @license     http://opensource.org/licenses/gpl-2.0.php GNU Public License
+ * @since       1.2
+ */
+
+// Exit if accessed directly.
+if ( ! defined( 'ABSPATH' ) ) exit;
+
+if ( ! class_exists( 'AffiliateWP_Affiliate_Area_Shortcodes' ) ) {
+
+	/**
+	 * Main plugin bootstrap.
+	 *
+	 * @since 1.0.0
+	 */
+	final class AffiliateWP_Affiliate_Area_Shortcodes {
+
+		/**
+		 * Holds the instance
+		 *
+		 * Ensures that only one instance of AffiliateWP_Affiliate_Area_Shortcodes exists in memory at any one
+		 * time and it also prevents needing to define globals all over the place.
+		 *
+		 * TL;DR This is a static property property that holds the singleton instance.
+		 *
+		 * @var object
+		 * @static
+		 * @since 1.0
+		 */
+		private static $instance;
+
+		/**
+		 * The version number of AffiliateWP
+		 *
+		 * @since 1.1
+		 */
+		private $version = '1.2';
+
+		/**
+		 * Main plugin file.
+		 *
+		 * @since 1.0.0
```

Comment:
The `$file` property was introduced in 1.2.
```suggestion
		 * @since 1.2
```